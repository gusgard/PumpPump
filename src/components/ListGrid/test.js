import { shallow } from 'enzyme';
import React from 'react';

import { ListGrid } from '@components';

const logo = { uri: 'https://...' };
const items = [{ id: 1, picture: logo }, { id: 2, picture: logo }];

it('renders correctly', () => {
  const wrapper = shallow(<ListGrid items={items} />);
  expect(wrapper).toMatchSnapshot();
});
