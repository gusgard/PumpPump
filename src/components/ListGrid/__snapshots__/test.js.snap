// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<FlatList
  data={
    Array [
      Object {
        "id": 1,
        "picture": Object {
          "uri": "https://...",
        },
      },
      Object {
        "id": 2,
        "picture": Object {
          "uri": "https://...",
        },
      },
    ]
  }
  disableVirtualization={false}
  horizontal={false}
  initialNumToRender={10}
  keyExtractor={[Function]}
  maxToRenderPerBatch={10}
  numColumns={3}
  onEndReachedThreshold={2}
  renderItem={[Function]}
  scrollEventThrottle={50}
  updateCellsBatchingPeriod={50}
  windowSize={21}
/>
`;
